src/main/java/SuBOT.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Scanner;
+
+public class SuBOT {
+    private static boolean stopFlag = false;
+    private static Task[] tasks = new Task[100];
+    private static int taskCount;
+
+    /**
+     * Process user's input command
+     * @param command Input command
+     */
+    public static void process(String command) {
+        String[] argv = command.split(" ");
+        int argc = argv.length;
```

Comment:
Perhaps you could use more descriptive names than argv and argc.

Reply:
I think `argc` and `argv` are already standard names here (at least in C++). It shouldn't be a problem.